workflows/src/test/kotlin/com/template/AppointmentRequestTests.kt

```diff
@@ -47,7 +47,7 @@ class AppointmentRequestTests {
     }
     @Test
     fun requestAppointmentTest() {
-        val availableDateFlow = CreateAppointmentDate(patientList, "06-07-2021")
+        val availableDateFlow = CreateAppointmentDate(alice.info.legalIdentities[0], bob.info.legalIdentities[0], "06-07-2021")
```

Comment:
Even if it makes using command line easier, I don't think it's a good idea to provide patients separately. Command line is not a primary way of using CorDapps and it's limited.